doc/classes/SurfaceTool.xml

```diff
@@ -29,6 +29,7 @@
 			</argument>
 			<description>
 				Adds an array of bones for the next vertex to use. [code]bones[/code] must contain 4 integers.
+				[b]Note:[/b] This must be called [i]before[/i] calling [method add_vertex]. Otherwise, it won't behave as desired.
```

Comment:
For these descriptions, I prefer to explain in more detail that this will be applied to the next vertex, if it is called after adding a vertex, it will be applied to the next vertex. If every vertex needs this and you fail to submit it for the first vertex, it may not be used at all. 

I don't like just saying it may not behave as desired.